ParkingFinder/handlers/parking_space.py

```diff
@@ -122,4 +126,80 @@ def post(self, user_id):
             })
 
 
+class ParkingSpaceRealTimeHandler(BaseHandler):
+
+    @coroutine
+    def get(self, user_id):
+
+        try:
+            access_token = self.get_argument('access_token', default=None)
+            # TODO: token validation
+            assert user_id
+            
+            realtime = yield ParkingSpaceService.initialize_waiting_user_location(user_id)
+
+            realtime_res = RealTimeMapper.to_record(realtime)
+            self.set_status(httplib.OK)
+            self.write({
+                'realtime': realtime_res
+            })
+
+        except AssertionError:
+            self.set_status(httplib.BAD_REQUEST)
+            self.write({
+                'error': "BAD REQUEST Invalid User Id"
+            })
+        except MissingArgumentError:
+            self.set_status(httplib.BAD_REQUEST)
+            self.write({
+                'error': "BAD REQUEST Location Information Is Incomplete"
+            })
+        except NotFound:
```

Comment:
because this user is wait for other people get his parking space, therefore, he will fetch for 5 mins, if return a error is not a good sign, if return not found, he can continue fetch until some one match him.